hutch_python/logging.yml

```diff
@@ -33,8 +33,22 @@ handlers:
 filters:
   ophyd_object_filter:
     (): hutch_python.log_setup.ObjectFilter
-    allow_other_messages: True
     level: DEBUG
+    # Allow these loggers through, even if noisy:
+    whitelist: ["pcdsdevices.interface", "hutch_python.utils"]
+    # Do not show these loggers (or ophyd device names):
+    blacklist: []
+    # Mark loggers / ophyd objects as "noisy" if they reach message levels
+    # above the given thresholds for the listed duration:
+    noisy_threshold_1s: 20
+    noisy_threshold_10s: 50
+    noisy_threshold_60s: 100
```

Comment:
I've been sitting here for a bit thinking about thresholds.

My first instinct is that we want these to be lower, but then I'm not quite sure. This includes debug logs, right? So I don't want it to trip too soon either.

Maybe these are already about right.

Reply:
I'm definitely not certain about these values at all. It's possible this indicates the proposal is using a flawed metric, but I can't think of a real alternative.

It only includes the debug level _if_ debug logs are shown:

```
            should_show = (
                record.levelno >= self._whitelist_all_levelno or
                name in self.object_names
            ) and name not in self.blacklist
```
A bit of confusing logic, for sure...

Reply:
if it's only console-visible logs then that makes it easier to think about, probably should be lower, but it also might mean that our logfiles still get flooded with debug messages

Reply:
The other thing is that the annoyance threshold for 1-line messages and 10-line tracebacks is a bit different

Reply:
Very true. Should "log message count" be inclusive of "number of lines in the message"? That seems appropriate...

Reply:
Maybe? But now we get really complicated because a single huge traceback would kill the log. But maybe it should?